### PR TITLE
update HTMLHint VS Code extension to official one

### PR DESCRIPTION
**Pull request by coliff**
Fixes #

## Proposed Changes

1. update HTMLHint VS Code extension to official one (the previous one is deprecated).

### Author/Contributor
- [ ] Add entry to the [CHANGELOG](https://github.com/oxsecurity/megalinter/blob/main/CHANGELOG.md) listing the change and linking to the corresponding issue (if appropriate)
- [ ] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

**Review by nvuillam — APPROVED** [review]
Good catch, thanks :)

File: docs/descriptors/html_htmlhint.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Use htmlhint in your favorite IDE to catch errors before MegaLinter !
...
-|  <img src="https://github.com/oxsecurity/megalinter/raw/main/docs/assets/icons/vscode.ico" alt="" height="32px" class="megalinter-icon"></a>  | [Visual Studio Code](https://code.visualstudio.com/) | [vscode-htmlhint](https://marketplace.visualstudio.com/items?itemName=mkaufman.HTMLHint)     | [![Install in VsCode](https://github.com/oxsecurity/megalinter/raw/main/docs/assets/images/btn_install_vscode.png)](vscode:extension/mkaufman.HTMLHint){target=_blank} |
+|  <img src="https://github.com/oxsecurity/megalinter/raw/main/docs/assets/icons/vscode.ico" alt="" height="32px" class="megalinter-icon"></a>  | [Visual Studio Code](https://code.visualstudio.com/) | [vscode-htmlhint](https://marketplace.visualstudio.com/items?itemName=HTMLHint.vscode-htmlhint)     | [![Install in VsCode](https://github.com/oxsecurity/megalinter/raw/main/docs/assets/images/btn_install_vscode.png)](vscode:extension/HTMLHint.vscode-htmlhint){target=_blank} |
```
Comment (nvuillam) [inline docs/descriptors/html_htmlhint.md:45]: If you see line 2 of this file, you'll see it has been entirely generated from descriptors :)

So you need to also update the descriptor, else the MD will be overwritten at the next doc build ^^

https://github.com/oxsecurity/megalinter/blob/9dcc80a15b91ab7e6f09752fe290cb48bcfab202/megalinter/descriptors/html.megalinter-descriptor.yml#L76

**Review by nvuillam — CHANGES_REQUESTED** [review]
Thanks for the catch
Plz check the comment :)

File: docs/descriptors/html_htmlhint.md (hunk 1 above)
Comment (coliff) [reply docs/descriptors/html_htmlhint.md:45]: ah ok - thanks! so I should close this PR and make a new one then I guess

**Review by nvuillam — APPROVED** [review]
Great, thanks for noticing and for the PR :)

**Comment by coliff** [comment]
> Great, thanks for noticing and for the PR :)

no problem. We just launched that official VS Code Extension yesterday. :-)

**Comment by nvuillam** [comment]
Congratulations ! :)

**Pull request merged**